dataverse-persistence/src/main/resources/Bundle_en.properties

```diff
@@ -3147,4 +3147,4 @@ validation.nonunique=The field must have a unique value.
 validation.value.not.allowed.in.controlled.vocabulary=Value "{0}" is not allowed. Please select one from the list.
 
 # geoboxTextAreaField.xhtml
-dataset.geobox.text.area.watermakr=Expected format\nLongitude Latitude\nExample \n21.017532 52.237049\n19.944544 50.049683
\ No newline at end of file
+dataset.geobox.text.area.watermakr=Expected format\nLongitude Latitude\nExample location: Maria Curie-Sk\u0142odowska University in Lublin\n22.54162 51.245524
```

Comment:
Actually I think that designated place for this translation is `geospatial_en.properties` and `geospatial_pl.properties`.
And instead of of using `Bundle` we should use `datasetFieldType.localeWatermark` (just like in case with watermarks for different fields)